src/qz/printer/status/CupsStatusHandler.java

```diff
@@ -45,21 +50,30 @@ private void parseXML(XMLEventReader eventReader) throws XMLStreamException {
         while(eventReader.hasNext() && running) {
             XMLEvent event = eventReader.nextEvent();
             switch(event.getEventType()) {
-                case XMLStreamConstants.START_ELEMENT:
+                case XMLStreamConstants.START_ELEMENT: {
                     StartElement startElement = event.asStartElement();
                     String qName = startElement.getName().getLocalPart();
                     if ("description".equalsIgnoreCase(qName)) {
                         isDescription = true;
+                        description = "";
                     }
                     if ("guid".equalsIgnoreCase(qName)) {
                         isGuid = true;
                     }
                     break;
+                }
+                case XMLStreamConstants.END_ELEMENT: {
+                    EndElement endElement = event.asEndElement();
+                    String qName = endElement.getName().getLocalPart();
```

Comment:
If you avoid double `qName` declaration, you can remove the `{}` brackets:

```java
String startElement = event.asEndElement().getName().getLocalPart();
```

```java
String endElement = event.asEndElement().getName().getLocalPart();
```

or even simpler:

```java
if ("description".equalsIgnoreCase(event.asEndElement().getName().getLocalPart())) {
    isDescription = false;
}
```